gnucash/gnome/gnc-plugin-page-invoice.c

```diff
@@ -274,6 +288,13 @@ static action_toolbar_labels invoice_action_labels[] =
     {NULL, NULL},
 };
 
+static action_toolbar_labels invoice_action_layout_labels[] =
+{
+    {"WindowsSaveLayoutAction", N_("_Save Layout for Customer Documents")},
+    {"WindowsResetLayoutAction", N_("_Reset Layout for Customer Documents")},
```

Comment:
Same here. To get `default` in there:
"Reset  Default Layout for Customer Documents"
or simply
"Reset  Default Layout"

The tooltip can do the rest.